new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java

```diff
@@ -321,7 +321,8 @@ public void testCreateMiiConfigMapDomain() {
       checkServiceCreated(managedServerPrefix + i, domainNamespace);
     }
 
-    int adminServiceNodePort = getAdminServiceNodePort(adminServerPodName + "-external", null, domainNamespace);
+    int adminServiceNodePort = getAdminServiceNodePort(adminServerPodName + "-external",
```

Comment:
I changed this method to have generic name. Its in PR https://github.com/oracle/weblogic-kubernetes-operator/pull/1648

Reply:
Ok, will resolve when its merged.